docs/guides/getting-hired.md

```diff
@@ -0,0 +1,53 @@
+# Getting Hired (for Developers)
+
+Thank you to [itinerare](https://github.com/itinerare) for providing the basis of this guide!

```

Comment:
This is thoughtful, but I think fine to omit-- and keeps the docs cleaner. At the end of the day, it's a group effort anyhow!